build/Versions.props

```diff
@@ -0,0 +1,34 @@
+﻿<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
+<Project>
+  <PropertyGroup>
+    <!-- This repo version -->
+    <VersionBase>1.0.0</VersionBase>
+    <PreReleaseVersionLabel>beta1</PreReleaseVersionLabel>
+  
+    <!-- Toolset -->
+    <DotNetCliVersion>1.0.1</DotNetCliVersion>
+    <RoslynToolsMicrosoftRepoToolsetVersion>1.0.0-alpha20</RoslynToolsMicrosoftRepoToolsetVersion>
+    <RoslynToolsMicrosoftXUnitLoggerVersion>1.0.0-alpha1</RoslynToolsMicrosoftXUnitLoggerVersion>
+    <RoslynToolsMicrosoftSignToolVersion>0.3.2-beta</RoslynToolsMicrosoftSignToolVersion>
+    <MicroBuildCoreVersion>0.2.0</MicroBuildCoreVersion>
+    <MicroBuildPluginsSwixBuildVersion>1.0.147</MicroBuildPluginsSwixBuildVersion>
+    <MicrosoftNetCompilersVersion>2.0.1</MicrosoftNetCompilersVersion>
+
+    <!-- Tests -->
+    <MicrosoftNETTestSdkVersion>15.0.0</MicrosoftNETTestSdkVersion>
+    <XUnitVersion>2.2.0</XUnitVersion>
+    <XUnitRunnerVisualStudioVersion>2.2.0</XUnitRunnerVisualStudioVersion>
+
+    <!-- desktop msbuild driver -->
+    <VSWhereVersion>1.0.47</VSWhereVersion>
+    <XUnitRunnerConsoleVersion>2.2.0</XUnitRunnerConsoleVersion>
```

Comment:
What about from VS 2017? I "guessed" that I might need them to run the tests from the VS test explorer.